### ISSUE TITLE
App Fills the Screen 

### ISSUE THREAD

**Issue by StephenUK1962** [marchyman/GeoTag#48]
The app fills the screen and cannot reduce the height or width of the window.

**Comment by marchyman** [comment]
Which version?  I recall that as being a bug that was fixed in the 5.2 release.

To resolve the issue you need to kill all of your settings and re-launching the app.  This requires a terminal command.

1) launch terminal
2) at the prompt enter this command
```sh
defaults delete org.snafu.GeoTag
```
3) quit terminal

**Comment by StephenUK1962** [comment]
Hi Marco,
 
The GeoTag version is 5.2
 
Sorry your fix did not work see screen shots attached.
<img width="1440" alt="Screenshot 2024-04-11 at 20 41 24" src="https://github.com/marchyman/GeoTag/assets/166653254/5e66e8d1-ac10-4a18-ae4b-04d1535a6561">
<img width="1440" alt="Screenshot 2024-04-11 at 20 38 37" src="https://github.com/marchyman/GeoTag/assets/166653254/4efb6a51-a264-4265-97b1-98d138837990">

  
Regards

**Comment by marchyman** [comment]
It looks like I misunderstood the issue.  How big is your screen?  You can find that in About This Mac, More Info Button.   The 5.2 version of GeoTag has its minimum window size set to 1200 wide by 900 high.  Is that too big for your display?

**Comment by StephenUK1962** [comment]
Hi

I have checked as you say the screen is 2560 x 1600 as per the screenshot attached
<img width="1440" alt="Screenshot 2024-04-12 at 08 09 20" src="https://github.com/marchyman/GeoTag/assets/166653254/ec8f613b-952e-4941-ad10-6c726d978102">

**Comment by marchyman** [comment]
Thanks for the info.  My *guess* is that 2560x1600 retina equates to 1280x800 window size.  I forgot about that reduction when converting retina -> normal sizes when I picked a minimum size.

I've created version 5.2.1.  The only difference between it and 5.2 is the window can be reduced to fit on your 13" MacBook pro.  If you'd like to give it a try you'll have to download it from my site.  It isn't available in the Mac App store.

https://www.snafu.org/GeoTag/GeoTag-5.2.1.dmg
